### PR TITLE
include progress documentation

### PR DESCRIPTION
**Pull request by KZzizzle**
## What do these changes do?

Includes information about logging progress

## Related issue number

Resolves TB issue https://github.com/ITISFoundation/osparc-issues/issues/369

## Checklist

- [x] I think the documentation is well written and up to date
- [ ] Do NOTE have very large media files (e.g. photos, videos, etc)
- [x] My spell-checker passes
- [ ] If you write a new section, i take ownership adding my user in .github/CODEOWNERS

### CODE REVIEW

File: docs/submission.md [hunk 1]
```diff
@@ -87,3 +87,17 @@ We need a **Command Line Interface (CLI)** to execute your code. This is the com
...
 * For Python, several options are well-described here: https://codeburst.io/building-beautiful-command-line-interfaces-with-python-26c7e1bb54df 
 
+## Showing Progress
+As a bonus feature, you may let the user know the progress of the code execution, if you are able to extract it during runtime. Examples of applicable cases include:
+* Knowledge of simulation time (e.g. the code simulates a process for 10 seconds, therefore knowledge of the second of simulation time gives #seconds*10%)
+* Knowledge of iterations required (e.g. for loop will iterate 100 times - each iteration is and additional 1% complete)
+
+To communicate this information, add a “print” or standard output statement to your code in the following formats:
+```
+"progress: 34%"
```
Comment (pcrespov) [inline docs/submission.md:97]: I would add a comment with a reference to where this is defined/tested ... so in the future we can see if the doc is outdated

```markdown
<!-- SEE test in
https://github.com/ITISFoundation/osparc-simcore/blob/6e2f71746a521fe0f48b582edb9af5b05ddb65b7/services/sidecar/tests/unit/test_log_parser.py#L39
-->
```

File: docs/submission.md [hunk 2]
```diff
@@ -87,3 +87,17 @@ We need a **Command Line Interface (CLI)** to execute your code. This is the com
 * For MATLAB, this is defined by the MATLAB Compiler
 * For Python, several options are well-described here: https://codeburst.io/building-beautiful-command-line-interfaces-with-python-26c7e1bb54df 
 
+## Showing Progress
+As a bonus feature, you may let the user know the progress of the code execution, if you are able to extract it during runtime. Examples of applicable cases include:
+* Knowledge of simulation time (e.g. the code simulates a process for 10 seconds, therefore knowledge of the second of simulation time gives #seconds*10%)
```
Comment (pcrespov) [inline docs/submission.md:92]: FYI: do you know that you can use latex formulas here? see https://upupming.site/docsify-katex/docs/#/supported

```markdown
$seconds * 10 %$
```

**Review by pcrespov — APPROVED** [review]
just some minor

File: docs/submission.md (hunk 2 above)
Comment (KZzizzle) [reply docs/submission.md]: thanks! makes sense

**Pull request merged**